package.json

```diff
@@ -223,7 +223,8 @@
   },
   "resolutions": {
     "@emotion/core": "10.0.28",
-    "react-hot-loader": "^4.13.0"
+    "react-hot-loader": "^4.13.0",
+    "csstype": "npm:csstype@3.0.11"
```

Comment:
This instructs yarn to resolve the alias from the npm registry. Check out the [yarn docs on this and other available protocols](https://yarnpkg.com/features/protocols#table).

I ended up using this protocol for the legacy resolutions (e.g. `emotion-theming`) to prevent installation of the legacy packages, essentially to replace them with the v11 equivalence.

In this case, it's not needed, since the resolution is for the version only. I did it this way purely for consistency.

Reply:
I think because we only ever pull from the npm registry, we probably shouldn't be using this. This line from that page is also worth noting (emphasis my own)

> While they work regardless of the context *we strongly recommend you to only use semver ranges* on published packages *as they are the one common protocol* whose semantic is clearly defined across *all package managers*

Useful context with regards to portability to pnpm later on.